net-misc/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+
+inherit gnome2 autotools
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2.2:=
+	>=dev-libs/dbus-glib-0.74
+	>=dev-libs/glib-2.32:2
+	>=net-misc/openfortivpn-1.2.0
+	gtk? (
+		app-crypt/libsecret
+		>=gnome-extra/nm-applet-1.0.5
+		>=x11-libs/gtk+-3.4:3
+	)
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
```

Comment:
Please add explicit version requirements from configure.ac, there might not be version this low in tree currently but not specifying versions would eventually lead to systems where package would fail to build because the requirement is not specific enough to pull the wanted version. TL;DR always write down version requirements from upstream at a minimum.